force-app/classes/src/ParameterizedMethodSpyCall.cls

```diff
@@ -8,6 +8,7 @@
 public class ParameterizedMethodSpyCall {
   private List<Object> params;
   public Object value;
+  public Exception error;
```

Comment:
**suggestion:** it could be an automatic property

It would protect it from being changed from the outside

```suggestion
  public Exception error {get; private set;};
```